### PR TITLE
Update registry.json for AAAU1

### PR DESCRIPTION
**Pull request by OffTheRicta**
Thanks.

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3116,5 +3116,19 @@
...
+ "poolId": "64f6bd547b5ec3c35e18d1b09956af8dd2e53b7da7a9b7f58f3c5580",  
+ "social": {                                                            
+   "twitter": "Richard79337561",
+   "facebook": "@adabankau",  
+   "instagram": " ",
+   "website": "https://adabank.com.au",
+   "youtube": " "
+ },
+ "telegram": "ADABankAU",                                             
+ "operatorName": "OffTheRicta",
```
Comment (Bez625) [inline registry.json:3131]: Please change this to either your real full name, real first name or leave blank

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
@OffTheRicta Please can you rebase this off master? I haven't been able to resolve the conflicts manually here

**Comment by OffTheRicta** [comment]
> @OffTheRicta Please can you rebase this off master? I haven't been able to resolve the conflicts manually here

Not sure how to do that. Do I start again?

**Comment by Bez625** [comment]
> > @OffTheRicta Please can you rebase this off master? I haven't been able to resolve the conflicts manually here
> 
> Not sure how to do that. Do I start again?

Starting again is an option or you can follow these steps https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a . Either should work

**Comment by Bez625** [comment]
Closing this PR as new PR opened.

**Pull request closed without merging**